flipperdevices/Flipper-Android-App
# Position bubble on key sub-ghz emulate and update hold modifier by new api

**Background**

* Deprecated point input API
* Not correct bubble 

**Changes**

* Previews
* Minus bubble cooridnate by Y from status bar
* New API hold

**Test plan**

* Try emulate single subghz and see corect display bubble

* Try hold for emulate
